### ISSUE TITLE
nvda not reading Bitdefender interface

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#17297]
### Steps to reproduce:
Where I'm working now, the antivirus is Bitdefender
When I open the program interface, NVDA announces the entire interface, but when I try to navigate with tabs or arrows, NVDA goes silent.
I think the problem is with the software, but I would like to ask NV Access to investigate.
I'll try to ask the company responsible for the software for support. If anyone knows a more accurate contact, I'd appreciate it.
### Actual behavior:
when navigating with arrows or tab nvda is silent

### Expected behavior:
nvda should be able to read the interface when navigating with tab or arrows

#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-34276

#### Windows version:
11.0.22621.4317
#### Name and version of other software in use when reproducing the issue:
Bitdefender epconsole endpoint security 7.9.16.441

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
[bitdefender.txt](https://github.com/user-attachments/files/17383447/bitdefender.txt)

**Comment by seanbudd** [comment]
There's nothing we can do about this. Please message the authors of the software
https://community.bitdefender.com/en/discussion/76317/make-bitdfender-ui-accessible

**Comment by Adriani90** [comment]
Closing as cant fix, and there is no upstream plattform to track external bugs with bit defender developers.
